### DISCUSSION TITLE
High Level Overview of Life Itself "Theory of Change" / Strategy

### DISCUSSION THREAD

**Discussion by rufuspollock** [life-itself/community#878] [General]
I created this diagram to lay out the "stepping down" from the highest level outcome to day to day activities.

This is part of the work to created a major new version "v3" of Life Itself Strategy (see #196).

See also previous diagram in https://lifeitself.org/notes/strategy-macro-to-micro-2023

![life-itself-strategy-2023-10-17](https://github.com/life-itself/community/assets/180658/187b4f73-3675-47f8-9950-9c1cf04f7fe3)

### Phase transition (between paradigms) for individuals and groups - aka Boiling the Water v2

![image](https://github.com/life-itself/community/assets/180658/751f044c-0164-4185-9268-79ae2b785f86)

[Source](https://app.excalidraw.com/s/9u8crB2ZmUo/8F30hhvlTcl)

### Boiling the water: zapping vs heating

![image](https://github.com/life-itself/community/assets/180658/a097fc96-b719-42fe-acbb-9ecf1d2a70fc)

[Source](https://app.excalidraw.com/s/9u8crB2ZmUo/ASUvwPJZZFN)

### Sanghas and Pockets

![image](https://github.com/life-itself/community/assets/180658/910d6397-d796-4db1-bcec-104361e2e92f)

[Source](https://app.excalidraw.com/s/9u8crB2ZmUo/8F30hhvlTcl)

**Comment by matthewmccarthy11** [comment]
Perhaps getting a sense for how research might be helpful as it relates to this diagram, especially 'how to support this transition', can be the aim of the friday meetings? @asimong curious what you think here

**Reply by asimong** [reply]
Thanks for pointing the question at me, @matthewmccarthy11 

And yes, "how to support the transition" is a good challenge.
The diagram is good, nothing wrong with it at all. As always, though, I'll point to the sense that the diagram illustrates one particular "healing narrative". As I keep on saying, all (well, nearly all) healing narratives are good, but not all healing narratives are effective for everyone. And some suit only a select few. Maybe we're in [Theory of Change](https://en.wikipedia.org/wiki/Theory_of_Change) territory here? I'm not advocating that particular approach, just suggesting, in a similar spirit, that the aim is to dig deep and map out what *exactly* is needed for the transitional changes to happen. Let's be skeptical as well as hopeful; critical as well as appreciative.

**Reply by rufuspollock** [reply]
Great question. Here's a partial answer: an extension I did a few months ago (but not sure i shared?) of the [strategy diagram shared in february](https://lifeitself.org/notes/strategy-macro-to-micro-2023). (NB: above strategy diagram is a more detailed evolution of that earlier one but has similar structure).

![image](https://github.com/life-itself/community/assets/180658/1072a52d-aa88-4f91-9ffa-520ef2a1fd7f)

**Comment by asimong** [comment]
So, maybe one way of going further would be to look at any other models we can get our hands on of this kind of transitional change. Maybe we can best start by helping each other unpack and lay out the "healing narratives" that sit, only partially conscious, in our own individual minds? Embody some critical friendship, point out the weak spots, the implausible spots, the "yes ok but that's never going to actually happen" points in each other's stories, in the spirit, not of criticism but of loving what is really possible, and saving each other from wasting our time pursuing dead ends.

And whenever there is a narrative that looks plausible, but not detailed enough to test out, to help each other to flesh them out. "How is that meant to work, in practice?"

AND, to look at the ideas and models of the many other groups around the world that are trying to do something in the same direction; to bring their insights to bear on "our" model and ours on theirs.

**Reply by asimong** [reply]
AND (repetition warning here!) to build something like a wiki to document all of this so that as new people catch on they don't have to reinvent their own training wheels.

**Reply by rufuspollock** [reply]
@asimong yes - i think research on models for this kind of transitional change would be v useful. This could range from historical analogies / examples to academic research on onto-social transitions.

**Comment by matthewmccarthy11** [comment]
Yeah I think that's a great idea. To start with one of the things that I've been considering lately as it relates to this question and one of our projects (cohere), it's this idea of a 'transformational space'-  I put together [this google document](https://docs.google.com/document/d/1u6EGQIMq6VQrWdiW4VvjcoRlRrFYx0tnlS2Z6T1N8Uw/edit) which provides some context (Maybe it already exists, although couldn't spot it. Also, I don't think 'catalyst' captures the full range of the process) I was thinking about sending it out but haven't yet, so feedback there would also be helpful.  'Transformational space 'is defined as: 'the space within a (complex) system which aims to transform the system'. The whole document might be a bit abstract, but I find it to be a potentially meaningful way to consider 'systems change'. The question that it is concerned with is: how does a (complex adaptive) system generate its own transformation? --> within a 'transformational space' which both generates transformation (new processes/ new ways of being?) but also has to exist 'within' the existing system, and the term 'liminal space' which is often used is interesting to consider here. In a way, the 'liminal space' is not just a **time** 'between worlds' but also two worlds **happening at once** or a time of coexisting worlds, even though this is a simplification. Would love to get some feedback on that though, since it seems to make sense in my mind. 

Also just want to emphasize here, which is included in the doc above, that the term 'system' is inherently vague when we consider things like 'systems change'

**Reply by asimong** [reply]
@matthewmccarthy11 thanks, and I've commented and made some small suggestions on that "Cohere+" Google Doc. I'd be interested to talk over some of the points I've raised. In particular, how do we think of "change", particularly in relation to "goal" in a complex system?

Separately, to me moving from "system" to "space" doesn't give me any benefit. To me, the feel of "space" is simpler, and therefore is more prone to leading to simplistic ways of thinking about systems. Rather, I'd like to see us using the word "system" with qualifications to indicate the boundaries of what we mean and don't mean.

**Reply by asimong** [reply]
Yes, completely agree that "system" is inherently vague — but it's always an option to clarify what *you* mean by "system" rather than choosing another word, unless you're sure that the other word is actually more in line with your concept, and communicates your intent better to your audience.

**Reply by asimong** [reply]
> how does a (complex adaptive) system generate its own transformation

Doesn't it just depend on the nature and qualities of the system? Some systems are resistant to transformation; but then I guess they don't count as adaptive. What comes to mind is Beer's VSM, which seems to detail what aspects of the system help to motivate or guide transformation … ? (Help, please, someone who is more familiar with VSM!) Or you could go with some analysis of what a system requires in order to count as “adaptive” …

**Reply by martinprogrammer** [reply]
@matthewmccarthy11  
I think that system is a good word.  
It can be a physical system (not vague) or a conceptual, or a contextual... Then you have a hierarchy of systems, an eco system (emphasising their interconnectivity)...  

Within a scope of a system - you don't need to know the intricacies of its working in order to decide whether it works or not.  That's done on the basis of its capability to achieve its intended goals and objectives.  You find out about those in the specs of the system or reverse-engineer their specs from the context of the higher level system.

In order to change it (for the purpose of this convo) we can either alter its behaviour (to align with goals better) or change it organisationally (to optimise it in terms of efficiency..).

Correct operation of sub-systems don't guarantee that the higher level system will operate well.  Therefore, the top level systems are the ones that need to be considered and worked backwards.

In terms of "changes" to a system - i think an option is to build alternative systems that would interface correctly into inter-dependent systems, so this "transformation" turns into "adaption".  

My thinking is whenever you want to change something profoundly, but still basing it on the previous version, will inevitably be architected sub-par.

Let me know if this is helpful ;-)
